zk_toolbox/crates/zk_supervisor/Cargo.toml

```diff
@@ -29,3 +29,4 @@ futures.workspace = true
 types.workspace = true
 serde_yaml.workspace = true
 zksync_basic_types.workspace = true
+sqruff-lib = { git = "https://github.com/quarylabs/sqruff", version = "0.18.2" }
```

Comment:
any reason to not use crate from crates.io?

Reply:
also, did you consider https://crates.io/crates/sqlformat? It has much more downloads

Reply:
Yes! They still didn't publish 1.18.2 (I already asked on their slack channel) so I had to import it like that.
I did try sqlformat but we weren't happy with the formatted queries.